app/services/spree_shopify_importer/data_parsers/stock_items/base_data.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module SpreeShopifyImporter
+  module DataParsers
+    module StockItems
+      class BaseData
+        CONTINUE = 'continue'
+        def initialize(spree_stock_location, inventory_level)
+          @spree_stock_location = spree_stock_location
+          @inventory_level = inventory_level
+        end
+
+        def variant_data_feed
+          SpreeShopifyImporter::DataFeed
+            .where(shopify_object_type: 'ShopifyAPI::Variant')
+            .where('data_feed like ?', "%inventory_item_id\":#{@inventory_level.inventory_item_id}%").first
+        end
+
+        def variant
+          Spree::Variant.find(variant_data_feed.spree_object_id)
+        end
+
+        def stock_item_attributes
+          {
+            stock_location_id: @spree_stock_location.id,
+            variant_id: variant.id
+          }
+        end
+
+        def backorderable?
+          @backorderable = JSON.parse(variant_data_feed.data_feed)['inventory_policy'] == CONTINUE
```

Comment:
why do you need to save `JSON.parse(variant_data_feed.data_feed)['inventory_policy'] == CONTINUE` into `@backorderable`, where are you using it later?